source/tests/xtest_complete_workflow.py

```diff
@@ -1,164 +0,0 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Having an API driven test for our key application workflows is a good automated regression test.  Is this moved somewhere?

Reply:
This is the test_complete_workflow.py file in the e2e directory.